src/anomalib/cli/cli.py

```diff
@@ -185,20 +195,21 @@ def add_export_arguments(self, parser: LightningArgumentParser) -> None:
         self._subcommand_method_arguments["export"] = added
         add_openvino_export_arguments(parser)
         self.add_arguments_to_parser(parser)
-        self.add_default_arguments_to_parser(parser)
 
     def add_hpo_arguments(self, parser: LightningArgumentParser) -> None:
         """Add hyperparameter optimization arguments."""
         parser = get_hpo_parser(parser)
 
     def add_benchmark_arguments(self, parser: LightningArgumentParser) -> None:
         """Add benchmark arguments to the parser."""
-        parser.add_argument("--config", type=Path, help="Path to the benchmark config.", required=True)
+        parser.add_argument("--benchmark_config", type=Path, help="Path to the benchmark config.", required=True)
```

Comment:
why we change `config` to `benchmark_config`? from the first glance it looks a bit verbose to me. I might prefer `config`

Reply:
It conflicts with the LightningArgparse's config command. I am not sure what's the best approach here. I can revert this to config but this might mean some additional checks that might pollute the implementation. Once we have pipelines this command might change. In that case should we even support hpo and benchmarking from the cli now? I am expecting the new command to be `anomalib pipelines benchmark ...` and `anomalib pipelines hpo ...`